sphinx/writers/latex.py

```diff
@@ -552,6 +575,15 @@ def __init__(self, document, builder):
                            builder.config.language)
 
         # set up multilingual module...
+        if self.elements['latex_engine'] == 'pdflatex':
+            if ('T2A' in self.elements['fontenc'] and
+                not self.babel.uses_cyrillic()):
+                self.elements['substitutefont'] = '\\usepackage{substitutefont}'
+                self.elements['sphinxpkgoptions'] += ',cyrnocyr'
+            if 'LGR' in self.elements['fontenc']:
+                self.elements['substitutefont'] = '\\usepackage{substitutefont}'
+            else:
+                self.elements['textgreek'] = ''
```

Comment:
This way no dependency is added to existing Sphinx projects only needing T1 encoding (i.e. all (non-Cyrillic) projects which did not have Unicode Greek letters in their text, previously rendered via TeX math font)